spec-integration/actions/deleteObject.json

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
Does something populate the SF instance with this data before the integration tests delete it?